examples/step-33/step-33.cc

```diff
@@ -266,16 +266,16 @@ namespace Step33
     // numerical flux function to enforce boundary conditions.  This routine
     // is the basic Lax-Friedrich's flux with a stabilization parameter
     // $\alpha$. It's form has also been given already in the introduction:
-    template <typename InputVector>
+    template <typename InputVector, typename number>
```

Comment:
We typically use uppercase first letters for template arguments, i.e., use Number